bin/populate_wtsi_irods_groups.pl

```diff
@@ -72,7 +77,7 @@
              exit 0;
            },
            'logconf=s'               => \$log4perl_config,
-           'study=i'                 => \@study_ids,
+           'study_id=i'              => \@study_ids,
```

Comment:
NB: also makes a breaking change to the CLI options

Reply:
Ah - I think not as `Getopt::Long` is clever enough to allow truncated variants of such command line options if not ambiguous i.e. ``` populate_wtsi_irods_groups.pl --dry-run  --study 86``` worked as before .